setup.py

```diff
@@ -58,6 +58,11 @@ def read_file(filename, encoding='utf8'):
 else:
     install_requires.append('mock')
 
+if sys.version_info < (2, 7, 9):
```

Comment:
They were already being pulled in `acme`, so by including them here you do not change anything!

So while still you're totally missing my point I did check it myself:

```
jakub@iks:~/dev/letsencrypt/letsencrypt/letsencrypt$ git grep 'import requests'
tests/validator_test.py:import requests
validator.py:import requests
jakub@iks:~/dev/letsencrypt/letsencrypt/letsencrypt$ grep 'requests.' validator.py
            response = requests.get(url, headers=headers, allow_redirects=False)
            response = requests.get(url, allow_redirects=False)
        headers = requests.get("https://" + name).headers
```

So I'm fine with keeping changes to the main  `setup.py`, but for reasons other than you mention. (AFAIK `validator.py` is a dead code - it's not used anywhere)